src/pages/trade-types/multiplier/_available-trades.js

```diff
@@ -0,0 +1,246 @@
+        transform-origin: bottom left;
+        box-shadow: 0 0 20px 0 rgba(0, 0, 0, 0.05);
+        ${(props) => {
+        if (props.active_tab === props.name)
+            return css`
+                    background-color: #ffffff;
+                    box-shadow: 0 0 20px 0 rgba(0, 0, 0, 0.05);
+                `
+    }}
+    }
+    @media ${device.tabletL} {
+        width: 100%;
+        height: 6rem;
+        min-width: unset;
+        padding-right: 0;
+    }
+`
+const TabMarginIcon = styled.img`
+    min-width: 16px;
+    ${(props) => {
+        if (props.active_tab === props.name)
+            return css`
+                margin-left: 16px;
+            `
+    }}
+`
+const TabOptionIcon = styled.img`
+    min-width: 16px;
+    ${(props) => {
+        if (props.active_tab === props.name)
+            return css`
+                margin-left: 16px;
+            `
+    }}
+`
+
+const ContentWrapper = styled.div`
+    width: 100%;
+    max-width: 99.6rem;
+    display: block;
+    background: #ffffff;
+    border-radius: 0.15em;
+    box-shadow: 0 0 20px 0 rgba(0, 0, 0, 0.1);
+
+    @media ${device.tabletL} {
+        padding: 0 2rem;
+    }
+`
+const CardHeader = styled(Header)`
+    @media ${device.tabletL} {
+        font-size: 1.75rem;
+        line-height: 16px;
+    }
+`
+const Card = ({ display_name, active_tab, onTabChange, name }) => {
+    return (
+        <CardContainer name={name} active_tab={active_tab} onClick={() => onTabChange(name)}>
+            <Flex height="fit-content" jc="flex-start" ai="center">
+                {active_tab === 'Forex' && (
+                    <TabMarginIcon
+                        src={ForexIcon}
+                        alt="margin icon"
+                        name={name}
+                        active_tab={active_tab}
+                    />
+                )}
+                {active_tab === 'Synthetic Indices' && (
+                    <TabOptionIcon
+                        src={SyntheticIcon}
+                        alt="option icon"
+                        name={name}
+                        active_tab={active_tab}
+                    />
+                )}
+                <CardHeader as="h4" type="sub-section-title" width="auto">
+                    {display_name}
+                </CardHeader>
+            </Flex>
+        </CardContainer>
+    )
+}
+
+Card.propTypes = {
+    active_tab: PropTypes.string,
+    display_name: PropTypes.object,
+    name: PropTypes.string,
+    onTabChange: PropTypes.func,
+}
+
+class AvailableTrades extends React.Component {
+    state = {
+        active_tab: 'Forex',
+    }
+    handleTabChange = (new_tab) => {
+        if (new_tab === this.state.active_tab) return
+        this.setState({ active_tab: new_tab })
+    }
+    render() {
+        const { display_title, Forex, SyntheticIndices } = this.props
+        return (
+            <StyledSection>
+                <StyledHeader size="var(--text-size-header-1)" align="center">
```

Comment:
`type`? can omit `size` if type is sufficient. We should add `as` for semantics as well.

Reply:
Looks like type can't fit in here...but we can add `as` though 👌